src/gui/addnewtorrentdialog.cpp

```diff
@@ -515,14 +515,18 @@ void AddNewTorrentDialog::displayContentTreeMenu(const QPoint &)
     QMenu *menu = new QMenu(this);
     menu->setAttribute(Qt::WA_DeleteOnClose);
 
+    QMenu *subMenu;
```

Comment:
Since it now doesn't always represent a submenu, I would rename it to something like `priorityMenu`.

Reply:
That's exactly what I wasn't sure of in this approach, thanks!